jsx/Form.js

```diff
@@ -1160,8 +1208,8 @@ TimeElement.propTypes = {
 TimeElement.defaultProps = {
   name: '',
   label: '',
-  value: '',
-  id: '',
+  value: null,
+  id: null,
```

Comment:
not in **24.1**

_same comments apply to TimeElement as were written for DateElement_

to include as **CBIG override** with plans to push to **main** in some capacity